app/views/wallet/TransactionView/SendMobPanel/SendMobForm.tsx

```diff
@@ -150,6 +153,33 @@ const SendMobForm: FC = () => {
     },
   ];
 
+  const handleChecked = () => {
+    setIsChecked(!isChecked);
+  };
+
+  const saveToContacts = async (alias: string, recipientAddress: string) => {
+    const randomColor = () => {
+      const RANDOM_COLORS = ['#8B35E0', '#1F639A', '#EAA520', '#15A389', '#8D969D', '#D82E26'];
+      return RANDOM_COLORS[Math.floor(RANDOM_COLORS.length * Math.random())];
+    };
+
+    const result = await assignAddressForAccount({
+      accountId: selectedAccount.account.accountId,
+    });
+
+    contacts.push({
+      abbreviation: alias[0],
```

Comment:
Can `alias` be more than one word? If so, the abbreviation needs to reflect that.